### DISCUSSION TITLE
SkiaSharp-Packages

### DISCUSSION THREAD

**Discussion by TopperDEL** [unoplatform/uno#8954] [Q&A]
If I want to render something with Skiasharp, which packages are the right ones?`

My solution consists of a Mobile-Project and a Windows-WinUI-Project. For the windows-Project, I would use "SkiaSharp.Views.WinUI". As far as I can tell this would include Skiasharp itself, right? But which one would be the right one for the Mobile-Project targeting iOs, Android and macOS? Is it the WinUI, the Uno or the MAUI-views-nuget?
the Uno-One seems to include Uno.UI, which is wrong here. WinUI seems to only target Windows - so it must be the "SkiaSharp.Views.Maui.Core"-package?

**Comment by jeromelaban** [comment]
The [SkiaSharp for WinUI Packages](https://github.com/mono/SkiaSharp/pull/2042) are not yet available. They are being worked on by Microsoft and should be available in the next few days.

**Reply by TopperDEL** [reply]
Ah, nice! Will this one be for all heads, then?
